pkg/config/config_test.go

```diff
@@ -65,6 +65,44 @@ var _ = Describe("Config", func() {
 			Expect(err).To(HaveOccurred())
 		})
 
+		It("Assuming correct config file - all multicast", func() {
```

Comment:
Indentation in this test code (tabs vs whitespaces)

Reply:
I'm using the same indentation of all other tests in this file